java/mapreduce/src/java/org/apache/orc/mapred/OrcMapredRecordReader.java

```diff
@@ -69,14 +69,19 @@ public OrcMapredRecordReader(RecordReader reader,
   }
 
   protected OrcMapredRecordReader(Reader fileReader,
-                                  Reader.Options options) throws IOException {
+      Reader.Options options) throws IOException {
```

Comment:
Could you keep the original style here? This looks inconsistent with the other existing one at line 64 and with your new code at line 77.